Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -20,34 +33,150 @@ final class ReceiptRenderer: UIPrintPageRenderer {
     }
 
     override func drawHeaderForPage(at pageIndex: Int, in headerRect: CGRect) {
-        let pageNumberString = NSString(string: "Order receipt. Page \(pageIndex + 1)")
-        pageNumberString.draw(in: headerRect, withAttributes: attributes)
+        guard let storeName = parameters.storeName else {
+            return
+        }
+
+        let receiptTitle = String.localizedStringWithFormat(Localization.receiptFromFormat,
+                                                            storeName) as NSString
+
+        receiptTitle.draw(in: headerRect, withAttributes: headerAttributes)
     }
 
-    override func drawContentForPage(at pageIndex: Int, in contentRect: CGRect) {
-        let printOut = NSString(string: "Total charged: \(parameters.amount / 100) \(parameters.currency.uppercased())")
+    override func drawFooterForPage(at pageIndex: Int, in footerRect: CGRect) {
+        guard let emv = parameters.cardDetails.receipt else {
+            return
+        }
+
+        /// According to the documentation, only `Application name` and `AID`
```

Comment:
Please add a link to https://stripe.com/docs/terminal/checkout/receipts#custom

Reply:
The link is two lines below that already 😉

Reply:
LOL Sorry!